frontend/src/scenes/data-warehouse/dataWarehouseOverviewLogic.ts

```diff
@@ -0,0 +1,114 @@
+import { actions, afterMount, kea, listeners, path, reducers, selectors } from 'kea'
+import { loaders } from 'kea-loaders'
+import { lemonToast } from '@posthog/lemon-ui'
+import api, { PaginatedResponse } from 'lib/api'
+
+import { ExternalDataSource } from '~/types'
+import type { dataWarehouseOverviewLogicType } from './dataWarehouseOverviewLogicType'
+
+export const dataWarehouseOverviewLogic = kea<dataWarehouseOverviewLogicType>([
```

Comment:
Could we rename this to `dataWarehouseSceneLogic` to keep consistent with other scene logics please?